.github/workflows/stale_issue_pr.yaml

```diff
@@ -0,0 +1,35 @@
+name: 'Stale issue & PR handler'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@main
+        id: stale
+        with:
+          ascending: true
+          close-issue-message: 'Issue closed due to inactivity.'
+          close-pr-message: 'Pull request closed due to inactivity.'
+          days-before-close: 10
+          days-before-stale: 30
+          stale-issue-label: stale
+          stale-pr-label: stale
+          # Not stale if have this labels 
+          exempt-issue-labels: 'bug'
+          exempt-pr-labels: 'bug'
```

Comment:
couldn't find more labels to add here, feel free to suggest more.

Reply:
We need to add `enhancement`. I will go through the open issues and make sure all issues are labelled correctly.

Reply:
We need to add enhancement. I will go through the open issues and make sure all issues are labelled correctly.

Reply:
added `enhancement` label to exempt